build/pr-code-cleanup.yml

```diff
@@ -70,7 +70,7 @@ jobs:
 
         if ($baseCommitHash -ne $headCommitHash) {
           Write-Output "Running code cleanup on commit range $baseCommitHash..$headCommitHash in pull request."
-          dotnet regitlint -s src/Steeltoe.All.sln --print-command --skip-tool-check --jb --dotnetcoresdk=$(dotnet --version) --jb-profile="Steeltoe Full Cleanup" --jb --properties:Configuration=Release --jb --verbosity=WARN -f commits -a $headCommitHash -b $baseCommitHash
+          dotnet regitlint -s src/Steeltoe.All.sln --print-command --skip-tool-check --jb --dotnetcoresdk=$(dotnet --version) --jb-profile="Steeltoe Full Cleanup" --jb --properties:Configuration=Release --jb --verbosity=WARN --jb --properties:NuGetAudit=false -f commits -a $headCommitHash -b $baseCommitHash
```

Comment:
Does this PR set NuGetAudit=false on all builds?

Reply:
No, only when running code cleanup.

Reply:
During regular builds, a warning is logged during `dotnet restore`. Code cleanup runs that internally, while warn-as-error is true, causing its build to fail.